eng/tools/generate-static-index/templates/matthews/partials/affix.tmpl.partial

```diff
@@ -0,0 +1,17 @@
+{{^_disableContribution}}
```

Comment:
Are all these files the support files for the generated pages? If so we should figure out a plan to keep these files in-sync across the repo's as well once we get everything using the same index generation.

Reply:
Right, these are static web templates which we use for landing page(Java, .NET, CPP). Things under eng/tools/generate-static-index/ are the same besides of the docfx.json and main.js. It contains some lang specific info. 
Not very easy to pull the common stuff out in common packages.

Reply:
OK I'd like to see if we can make those language specific stuff into the language-settings.ps1 file in some way so that we can still share the majority.